lib/collection/proxy-config.js

```diff
@@ -40,8 +46,9 @@ _.inherit((
      * @example <caption>Create a new ProxyConfig</caption>
      * var ProxyConfig = require('postman-collection').ProxyConfig,
      *     myProxyConfig = new ProxyConfig({
-     *          match: 'https://example*',
-     *          server: 'https://proxy.com',
+     *          host: 'proxy.com',
+     *          match: 'http+https://example.com/*',
```

Comment:
now, in ui it will be constructed. 
we are following similar to npm git url representation.